docs/tables/aws_organizations_account.md

```diff
@@ -0,0 +1,21 @@
+# Table: aws_organizations_account
+
+Contains information about AWS accounts that are members of the current organization.
+
```

Comment:
```suggestion

Note: Need to set AWS organization account IAM access key credential to query this table. 
```

Reply:
Agree that it's useful to ensure users know which IAM creds to use. I've added a bit more context based on the text from the AWS Organizations API docs:

> Note: This table can only be queried using IAM access key credentials for the AWS Organizations management account or a member account that is a delegated administrator for an AWS service.